build/LinuxDistrosNativeInstaller.props

```diff
@@ -1,22 +1,20 @@
 <Project>
   <ItemGroup>
     <LinuxDistrosNeedNativeInstaller Include="ubuntu.16.04-x64">
```

Comment:
Why do we still need these `ubuntu.XX.04-x64` entries?

Reply:
We should need only a single ubuntu leg to produce the .deb file.

Reply:
> however, continue to publish to all 3 Debian 'feeds'

I thought we were going to stop publishing to the Debian feeds.  We had a discussion about this a few weeks ago.

/cc @weshaggard

Reply:
@weshaggard 
@eerhardt 
@livarcocc 
To be clear, we are saying that I should completely remove the publishing to the debian feeds for all variants.

Reply:
@weshaggard
@eerhardt
@livarcocc
To be clear, we are saying that I should completely remove the publishing to debian feeds for all variants. Please advise.

Reply:
I think that's the consensus from the discussion.

I don't think you'd need to completely remove all the debian publish feed stuff in this change.  I would just remove the extra `LinuxDistrosNeedNativeInstaller` entries.  And then in a separate change we can remove the debian publish feed stuff.